docs/branches.md

```diff
@@ -13,55 +13,47 @@ branches. There is currently no concept of an active/current/checked-out branch.
-remind you that you may want to push the branch to some remote.
-
-When you pull from a remote, any changes compared to the current record of the
-remote's state will be propagated to the local branch. Let's say you run
-`jj git fetch --remote origin` and the remote's "main" branch has moved so its
-target is now ahead of the local record in `main@origin`. That will update
-`main@origin` to the new target. It will also apply the change to the local
-branch `main`. If the local target had also moved compared to `main@origin`
-(probably because you had run `jj branch set main`), then the two updates will be
-merged. If one is ahead of the other, then that target will be the new target.
-Otherwise, the local branch will be conflicted (see next section for details).
-
-<!-- TODO: Adjust this paragraph to the new defaults which were introduced in #2736 -->
-As of December 2023 Jujutsu tracks[^1] and fetches all branches by default, 
-which is confusing users coming from Git. To smoothen the transition branch 
-tracking was introduced. 
-
-### What does `git.auto-local-branch` actually do? 
+`<branch name>@<remote name>`, such as `jj new main@origin`.
+
+A remote branch can be associated with a local branch of the same name. It's
+sometimes called a tracking branch. When you pull from a remote, any changes
+compared to the current record of the remote's state will be propagated to the
+tracking local branch. Let's say you run `jj git fetch --remote origin` and the
+remote's "main" branch has moved so its target is now ahead of the local record
+in `main@origin`. That will update `main@origin` to the new target. It will also
+apply the change to the local branch `main`. If the local target had also moved
+compared to `main@origin` (probably because you had run `jj branch set main`),
+then the two updates will be merged. If one is ahead of the other, then that
+target will be the new target. Otherwise, the local branch will be conflicted
+(see the next "Conflicts" section for details).
+
+Most commands don't show the tracking remote branch if it has the same target as
+the local branch. The local branch (without `@<remote name>`) is considered the
+branch's desired target. Consequently, if you want to update a branch on a
+remote, you first update the branch locally and then push the update to the
+remote. If a local branch also exists on some remote but points to a different
+target there, `jj log` will show the branch name with an asterisk suffix
+(e.g. `main*`). That is meant to remind you that you may want to push the branch
+to some remote.
+
+By default, the default remote branch (e.g. `main@origin`) will be tracked
+automatically. You can use `jj branch track` command to track existing remote
+branches individually, or set `git.auto-local-branch = true` configuration to
+track new remote branches automatically.
+
+### What does `git.auto-local-branch = true` actually do?
 
 Jujutsu's fetch operations consist of several steps. First `jj git fetch`
```

Comment:
nit: s/operations/operation/ since we only have one builtin fetch operation right now (and the similar thing at google doesn't have these steps)